libraries/standard/mqtt/cbmc/stubs/NetworkInterfaceReceiveStub.c

```diff
@@ -0,0 +1,19 @@
+#include "mqtt.h"
+
+int32_t NetworkInterfaceReceiveStub( NetworkContext_t context,
+                                     void * pBuffer,
+                                     size_t bytesToRecv )
+{
+    __CPROVER_assert( context != NULL,
+                      "IotNetworkInterfaceReceive pConnection is not NULL." );
+
+    __CPROVER_assert( pBuffer != NULL,
+                      "IotNetworkInterfaceReceive pBuffer is not NULL." );
+    __CPROVER_havoc_object( pBuffer );
+
```

Comment:
Should we be asserting that pBuffer points to a buffer with at least bytesToRecv characters in it?

Reply:
How do we do that?